.github/workflows/generate_erd.yml

```diff
@@ -0,0 +1,88 @@
+name: Generate Models Graph
+
+on:
+  push:
+    paths:
+      - "backend/apps/**/models/**"
+      - "pyproject.toml"
+      - "README.md"
+  workflow_dispatch:
+
+jobs:
+  generate-model-graph:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.13"
+
+      - name: Install Poetry
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install poetry
+
+      - name: Install system dependencies
+        run: |
+          sudo apt update
+          sudo apt install -y graphviz
+
+      - name: Create and Populate .env File
+        run: |
+          touch backend/.env  # Ensure the file exists
+          echo "DJANGO_SENTRY_DSN=${{ secrets.DJANGO_SENTRY_DSN }}" >> backend/.env
+          echo "DJANGO_SECRET_KEY=${{ secrets.DJANGO_SECRET_KEY }}" >> backend/.env
+          echo "SECRET_KEY=${{ secrets.SECRET_KEY }}" >> backend/.env
+          echo "DJANGO_CONFIGURATION=Test" >> backend/.env
+          echo "GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}" >> backend/.env
+          echo "DJANGO_DB_PASSWORD=${{ secrets.DJANGO_DB_PASSWORD }}" >> backend/.env
+          echo "DJANGO_ALLOWED_HOSTS=*" >> backend/.env  # Example of a non-secret variable
```

Comment:
This doesn't look right. You shouldn't need the real secrets for building a model relations graph.